examples/UsbMidiSynth/UsbMidiSynth.ino

```diff
@@ -37,23 +37,49 @@ void loop() {
   while (USBMIDI.available()) {
 
     //Parse MIDI
-    u8 command=0, channel=0, pitch=0;
+    u8 command=0, channel=0, key=0, pitchbend=0, pblo=0, pbhi=0, velocity=0;
+    while(!(USBMIDI.peek() & 0b10000000)) USBMIDI.read(); //Drop bad commands
```

Comment:
Actually MIDI protocol allows sending only data bytes, in case the command is still the same, so it's not 'bad commands' :) To parse this, you have to keep track of the last MIDI command.

You could get data like (in hex): `90 40 30 40 00` for note 64 on, note 64 off. At the moment USBMIDI library doesn't provide data like this from MIDI over USB, as USB MIDI packets always contain the command byte. This is good to keep in mind when working with MIDI. For handling the case when only data bytes are received, I'd recommend using [MidiToUsb](https://github.com/BlokasLabs/USBMIDI/blob/master/src/midi_serialization.h#L24) class, to convert the raw MIDI stream into events you could easily handle. (the m_event field is defined in USB MIDI spec [on page 16](https://www.usb.org/sites/default/files/midi10.pdf))

Something like:

```cpp
// Global variable
MidiToUsb serializer;

// Inside loop
midi_event_t event;
if (USBMIDI.available() > 0 && serializer.process(USBMIDI.read(), event))
{
    // We have a full event that we can process.
    switch (event.m_event & 0x0f)
    {
    case 0x08:
    case 0x09:
        // Handle note on / off using event.m_data[...], 0 byte is MIDI command, 1 is note number, 2 is velocity
        break;
    }
}
```

Anyway, the comment should say instead 'Skipping to next command' to be more accurate. :)